public/js/map_direct.js

```diff
@@ -59,13 +59,13 @@ function submitEntry(obj, id) {
     let gasoline = Number(!($(".mapboxgl-popup-content").find('.gasoline img').hasClass('no-gas')));
     let diesel = Number(!($(".mapboxgl-popup-content").find('.diesel img').hasClass('no-gas')));
     let lpg = Number(!($(".mapboxgl-popup-content").find('.lpg img').hasClass('no-gas')));
-    validateCaptcha((token) => {
+    // validateCaptcha((token) => {
         let data = {
             "fuel_station": id,
             "gasoline": gasoline,
             "diesel": diesel,
             "lpg": lpg,
-            "captcha": token
+            // "captcha": token
         }
         $(obj).parent().parent().find('.popup_submit_text').html("VALIDADO");
```

Comment:
This approach is very dangerous. It is too bounded to markup. If a new layer is added/removed it will break.
Instead we should use a selector to navigate up on the tree to grab the desired element.

https://api.jquery.com/parent/#parent-selector